src/views/tutorial-view/index.tsx

```diff
@@ -145,7 +145,8 @@ function TutorialView({
 	const nextPreviousData = getNextPrevious({
 		currentCollection: collectionCtx.current,
 		currentTutorialSlug: slug,
-		nextCollectionInSidebar: tutorial.nextCollectionInSidebar,
+		nextCollectionInSidebar:
+			collectionCtx.current.nextCollection ?? tutorial.nextCollectionInSidebar,
```

Comment:
Let's actually handle this in the `server` file? So we could calculate `nextCollectionInSidebar` there.

Reply:
I can do that!

Reply:
Also I do think we'll need to adjust the href to be the dev portal path since the collection slug doesn't map to the actual path in developer. We have some helper functions, let me find them.

Reply:
[getCollectionSlug!](https://github.com/hashicorp/dev-portal/blob/63ace1b5b948b09158b8f9f4c469f71637101c0c/src/views/collection-view/helpers/get-slug.ts#L38-L53)

Reply:
The formatting is already handled in the helper [here](https://github.com/hashicorp/dev-portal/blob/main/src/views/tutorial-view/components/next-previous/helpers.ts#L57), so no need to change anything! (:

Reply:
Moved it to server.ts in this commit: https://github.com/hashicorp/dev-portal/pull/2215/commits/40166997ad7272eb89e591b442a288b7847fb2ed